### DISCUSSION TITLE
Q: Slew telescope to Panel - 3 buttom script command

### DISCUSSION THREAD

**Discussion by pzman3d** [Stellarium/stellarium#1595] [Q&A]
May I ask you some questions?

in the "Slew telescope to" Control Panel.

Center of the Screen .
Slew.
Sync.

3 function script command.

**Comment by gzotti** [comment]
https://stellarium.org/doc/head/group__telescopeControl.html

public slots are scriptable commands.

**Reply by pzman3d** [reply]
Thanks for your reply.

I have read this doc

But i mean is :

how to use script to run : 

1.Center of the Screen button > Get J2000
2.Sync Button > Apply RA+DEC calibration to MY Mount
3.Slew Button > Run Slew use RA+DEC not 

slewTelescopeToSelectedObject (const int idx) - need SelectedObject.
slewTelescopeToViewDirection (const int idx) - i want sync ToViewDirection not slew.
syncTelescopeWithSelectedObject (const int idx) - need SelectedObject.

i want to use 3 script command to auto run and center object (  i have make a soft can use Plate solover).
